src/routes/cart.route.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const router = express.Router();
+const {UserData} = require('../models/userData.model')
+const {Product} = require('../models/product.model') 
+const { authVerify} = require('../utls/authVerify')
+
+// Load Initial cart
+router.get('/', authVerify, async(req, res) => {
+    const {userId} = req.user
+    const userData = await UserData.findOne({userId: userId}).populate({
+        path: 'cartData',
+        populate: {
+            path:'product',
+            model: 'Product'
+        } 
+    })
+    console.log(userData.cartData)
+    const normalizedCartData = userData.cartData.map(item => {return {qty: item.qty, ...item.product._doc}})
+    res.json({status: "success", cartData: normalizedCartData})
+})
+
+router.route('/:productId')
+// Add to cart
+.post(authVerify, async(req, res) => {
```

Comment:
As you are using `authVerify` middleware everwhere you can write it on top of the file